pkgs/applications/editors/jupyter-kernels/iruby/update.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env sh
+
+# First, manually update Gemfile
```

Comment:
Can we also update the Gemfile in a script?
I'm not familiar with Ruby, but this API might work, or there might be something more appropriate.

`GET - /api/v1/versions/[GEM NAME]/latest.json`
https://guides.rubygems.org/rubygems-org-api/

Reply:
Okay, just did this in the latest commit!